_sass/minimal-mistakes/_pyos-grid.scss

```diff
@@ -135,20 +136,21 @@
         cursor: pointer;
     }
 
-    &:before {
-        content: "";
-        position: absolute;
-        z-index: -1;
-        top: -16px;
-        right: -16px;
-        background: #967eb6;
-        height: 32px;
-        width: 32px;
-        border-radius: 32px;
-        transform: scale(2);
-        transform-origin: 50% 50%;
-        transition: transform 0.15s ease-out;
-    }
+
+    // &:before {
```

Comment:
Remove the block 😄

Reply:
@willingc 😆 you have caught me in my usual pattern of being nervous to sometimes delete things so i comment it out until i'm sure i want to really delete it! i'll go through and cleanup comments. this was me removing the circles on our people cards but i think i want to keep them for other content cards . so i'll make that fix locally!!